src/gui/locator/qgslocatorwidget.h

```diff
@@ -66,6 +66,11 @@ class GUI_EXPORT QgsLocatorWidget : public QWidget
      */
     void setMapCanvas( QgsMapCanvas *canvas );
 
+    /**
+     * \brief Set placeholder text for the line edit.
```

Comment:
```suggestion
     * \brief Set placeholder \a text for the line edit.
     * \since QGIS 3.36
```

(and then re-run sipify)